source/_components/device_tracker.bluetooth_tracker.markdown

```diff
@@ -27,6 +27,10 @@ device_tracker:
   - platform: bluetooth_tracker
 ```
 
+Configuration variables:
```

Comment:
Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration